docs/preview/02-Features/publishing-events.md

```diff
@@ -18,15 +18,15 @@ PM> Install-Package Arcus.EventGrid.Core
 
 ## Usage
 Adding simple Azure EventGrid publishing to your application only requires the following registration.
-> ⚠ Note that this way of registering requires the [Arcus secrect store](https://security.arcus-azure.net/features/secret-store) to retrieve the necessary authentication secrets to interact with the Azure EventGrid topic.
+> ⚠ Note that registering using non-managed identity authentication requires the [Arcus secrect store](https://security.arcus-azure.net/features/secret-store) to retrieve the necessary authentication secrets to interact with the Azure EventGrid topic.
```

Comment:
Should we somewhere make it clear that authentication using Managed Identity is the recommended / preferred way ?

Reply:
Done in https://github.com/arcus-azure/arcus.eventgrid/pull/249/commits/6e55afe169674d0122409989c36b4065a7f7715e